PrimeNumbers.py

```diff
@@ -0,0 +1,29 @@
+def is_prime(num):
+    if num <= 1: # 1 is not a prime number
+        print("Not a prime number")
+        return
+    for i in range(2, num): # loop to check range
```

Comment:
Missing empty line after a `IF` block, PEP8 convention. 

`i` is not a meaning full name.

Do we need to run the loop from `2 to num`? A number `N` can never be divided by any value greater than`N//2` i.e number `10` can't be divisible of any number greater than `10/2 = 5` and we can avoid checking values from `6 - 10` for number 10.